lib/utils/compose.js

```diff
@@ -530,10 +530,13 @@ const getPreviousRepos = (sdk, docker, logger, appID) =>
 						return registry.imageName;
 					});
 				});
+			} else {
+				return Promise.resolve([]);
 			}
```

Comment:
There's no need for the promise.resolve here

Reply:
Done